docs/releasenotes/release-1.126.md

```diff
@@ -1,34 +1,39 @@
+* Special shout-outs to @acpana, @Camila-B, @cheftako, @ericpang777, @gemmahou, @himanikh, @jasonvigil, @jingyih, @jsoref, @justinsb, @maqiuyujoyce, @nb-goog, @xiaoweim, @yuwenma, @ziyue-101 for their contributions to this release.
 
 ## Announcement 
 
-* Promoted `ControllerReconciler` and `NamespacedControllerReconciler` CRDs from v1alpha1 to v1beta1.
+* Config Connector system management CRDs `ControllerReconciler` and `NamespacedControllerReconciler` are promoted to Beta. See how to [configure the Controller manager rate limit](https://cloud.google.com/config-connector/docs/how-to/customize-controller-manager-rate-limit).
 
 ## New Beta Resources (Direct Reconciler):
 
-* `Placeholder`
+* [`BigQueryTransformConfig`](https://cloud.google.com/bigquery/docs/reference/datatransfer/rest/v1/projects.locations.transferConfigs)
 
-## New Fields:
+ * Manage the metadata needed to perform a Big Query data transfer.
 
-* `GkeHubFeatureMembership`
-  * Added `spec.configmanagement.management` field to enable Config Sync Auto Upgrade.
+* [`KMSKeyHandle`](https://cloud.google.com/bigquery/docs/reference/datatransfer/rest/v1/projects.locations.transferConfigs)
 
-* `IAMPolicyMember` and `IAMPartialPolicy`
-  * [Added](https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/3231) `spec.memberFrom.bigQueryConnectionConnectionRef` field to enable referencing the service account generated by the `BigQueryConnectionConnection` resource.
+ * Manage the provisioning of a CryptoKey.
 
 ## Modified Beta Reconciliation
 
-We migrated the following reconciliation from the TF-based or DCL-based controller to the new Direct controller to enhance the reliability and performance. The resource CRD is unchanged.
+We have added support for direct reconciliation to more resources, with opt-in behaviour. The API is unchanged. To use the direct reconciler, add the `alpha.cnrm.cloud.google.com/reconciler: direct` annotation to the corresponding Config Connector object. The following resources now have direct reconciliation support (and we list some of the issues that this fixes): 
+
+* [`GkeHubFeatureMembership`](https://cloud.google.com/kubernetes-engine/fleet-management/docs/fleet-concepts/fleet-features)
+
+  * Enable [ConfigSync Auto-upgrade](https://cloud.google.com/kubernetes-engine/enterprise/config-sync/docs/how-to/upgrade-config-sync) via `spec.configmanagement.management: MANAGEMENT_AUTOMATIC`
+
+## New Fields:
+
+* `IAMPolicyMember`
+
+  * Use Service Account from BigQueryConnectionConnection via `spec.memberFrom.bigQueryConnectionConnectionRef`. See [example](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/d2d887f6e416d329930b14d1e7ab99a3b4591ecb/config/samples/resources/iampolicymember/policy-member-with-bigqueryconnection-reference/iam_v1beta1_iampolicymember.yaml#L23) on IAMPolicyMember use BigqueryConectionConnection "cloudSQL"
```

Comment:
nit: There is a public doc of the same sample. Not sure if we want to link to that.

https://cloud.google.com/config-connector/docs/reference/resource-docs/iam/iampolicymember#policy_member_with_bigqueryconnection_reference

Reply:
ah, because the reference doc was not updated when I'm working on this. I'm fine to leave it to the source code (I started to introduce the link as a bonus for advanced users. They can find many examples.)